### ISSUE TITLE
E712 false positive

### ISSUE THREAD

**Issue by attilaolah** [PyCQA/pycodestyle#218]
The following code:

``` python
    @classmethod
    def query(cls, *args, **kw):
        """Exclude deleted entities."""
        args += cls.deleted == False,
        return super(Model, cls).query(*args, **kw)
```

results in

> E712 comparison to False should be 'if cond is False:' or 'if not cond:'

However, this is not a comparison to False. The `cls.deleted` is an instance of an `ndb.Property` subclass (Google App Engine's Datastore abstraction library) that overrides `__eq__` to return query parameters (filters) instead of boolean values.

I wonder if PEP8 should be less strict about boolean comparisons with `==` or is this just a side-effect of too much overloading in the `ndb` library, in which case this can be freely closed as _wontfix_.
